clientlibs/js/src/UpgradeClient.ts

```diff
@@ -3,14 +3,14 @@ import { Interfaces } from './identifiers';
 import setWorkingGroup from './functions/setWorkingGroup';
 import getAllExperimentConditions from './functions/getAllExperimentConditions';
 import {
-  IExperimentAssignment,
   IFeatureFlag,
   ISingleMetric,
   IGroupMetric,
   ILogInput,
   MARKED_DECISION_POINT_STATUS,
+  IExperimentAssignment2,
```

Comment:
Can we have proper version number in the type name ie v4

Reply:
code updated